doc/cephfs/posix.rst

```diff
@@ -47,3 +47,36 @@ POSIX semantics for various reasons:
   code.  The name of this hidden directory can be changed at mount
   time with ``-o snapdirname=.somethingelse`` (Linux) or the config
   option ``client_snapdir`` (libcephfs, ceph-fuse).
+
+Perspective
+-----------
+
+People talk a lot about "POSIX compliance," but in reality most file
+system implementations do not strictly adhere to the spec, including
+local Linux file systems like ext4 and XFS.  For example, for
+performance reasons, the atomicity requirements for reads are relaxed:
+processing reading from a file that is also being written may see torn
+results.
```

Comment:
@ukernel is this accurate?  are there other local fs diversions from posix worth noting?

Reply:
yes, it's accurate. I don't think of any other diversions